guides/common/modules/ref_examples-ldap-filters.adoc

```diff
@@ -11,8 +11,16 @@ As an administrator, you can create LDAP filters to restrict the access of speci
 | User1, User3 |(memberOf=cn=Group1,cn=Users,dc=domain,dc=example)
 | User2, User3 |(memberOf=cn=Group2,cn=Users,dc=domain,dc=example)
 | User1, User2, User3 | (\|(memberOf=cn=Group1,cn=Users,dc=domain,dc=example)(memberOf=cn=Group2,cn=Users,dc=domain,dc=example))
+| User1, User2, User3 | (memberOf:1.2.840.113556.1.4.1941:=cn=Users,dc=domain,dc=example)
 |====
 
+[NOTE]
+----
+Group `Users` is a nested group that contains groups `Group1` and `Group2`.
```

Comment:
"contains member groups" to be extra clear what we mean by that?

Reply:
Well I just wanted to say that Users is a nested group because it has other groups in it.